hivemq-edge/src/frontend/src/extensions/datahub/components/pages/PolicyEditor.tsx

```diff
@@ -39,11 +40,12 @@ const PolicyEditor: FC = () => {
   const { t } = useTranslation('datahub')
   const reactFlowWrapper = useRef(null)
   const [reactFlowInstance, setReactFlowInstance] = useState<ReactFlowInstance | null>(null)
-  const { nodes, edges, onNodesChange, onEdgesChange, onConnect, onAddNodes } = useDataHubDraftStore()
+  const { status, nodes, edges, onNodesChange, onEdgesChange, onConnect, onAddNodes } = useDataHubDraftStore()
   const edgeConnectStart = useRef<OnConnectStartParamsNode | undefined>(undefined)
-
   const nodeTypes = useMemo(() => CustomNodeTypes, [])
 
+  const isEditable = status !== DesignerStatus.LOADED
```

Comment:
question(non-blocking): should this one be computed via `useMemo`?

```suggestion
  const isEditable = useMemo(() => status !== DesignerStatus.LOADED, [status])
```